src/App.tsx

```diff
@@ -1,27 +1,72 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
+import fetchName from './helper/fetchName';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  // eslint-disable-next-line
-  console.log(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    clockName: 'FRUIT',
+    isClockVisible: true,
+  };
+
+  changeNameHandler = async () => {
+    this.setState({
+      clockName: await fetchName(),
```

Comment:
why are you fetching names all the time? just fetch them once, in `componentDidMount` and save all the values in state